flutter/alds/lib/mappage.dart

```diff
@@ -27,112 +27,92 @@ class AldsMapPage extends StatefulWidget {
 }
 
 class _AldsMapPageState extends State<AldsMapPage> {
-  // State variables
```

Comment:
Can you restore all these inline comments (for the rest of this file, too), please? I think they're pretty helpful, if not for us but also for Prof. Reiss when we hand this code off to him.